spec/credit_cards_spec.rb

```diff
@@ -15,9 +15,7 @@
 
   describe 'SaveCreditCard' do
     context 'when the response is enabled' do
-      before do
-        allow(ResponseToggler).to receive(:enabled?).with('save_credit_card').and_return(true)
-      end
+      before { ResponseToggler.enable('save_credit_card') }
```

Comment:
Hi guys! long time no see :smile:

Just passing by to warn about `ResponseToggler` :warning:. Calling `enable` in the tests will actually write to redis. Therefore, these settings changes will leak between specs. So, I recommend keeping the stubs as they were, for safer tests.